language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs

```diff
@@ -26,8 +26,9 @@ eslintVersion :: T.Text
 eslintVersion = "^6.8.0"
 
 -- Version of typescript-eslint for linting the generated code.
+-- 2.23 produces an error https://github.com/typescript-eslint/typescript-eslint/issues/2009
```

Comment:
This should actually say 2.32

Reply:
I can fix in a followup to get this one through

Reply:
just make it a suggestion and you save me having to open an editor

Reply:
i would have taken care of it ;)